.github/workflows/reusable-assemble.yml

```diff
@@ -62,6 +62,18 @@ jobs:
             exit 1
           fi
 
+      - name: Check for jApiCmp diffs
```

Comment:
we only run `assemble` above (not `check` or `build`), maybe add `assemble jApiCmp` above, or run `jApiCmp` on `assemble` (or even `compileJava`?) instead of on `check`